models/Search/Backend/Data.php

```diff
@@ -534,7 +534,7 @@ public function setDataFromElement(Element\ElementInterface $element)
 
         $pathWords = str_replace([ '-', '_', '/', '.', '(', ')'], ' ', $this->getFullPath());
         $this->data .= ' ' . $pathWords;
-        $this->data = 'ID: ' . $element->getId() . "  \nPath: " . $this->getFullPath() . "  \n"  . $this->cleanupData($this->data);
+        $this->data = 'ID: ' . $element->getId() . "  \n"  . $this->cleanupData($this->data);
```

Comment:
@aashan10 I think that's the only relevant change left, right? All the other changes have no more relevancy anymore and can be removed.

Reply:
Well, this and the message queue part I think. These two are the only things needed for resolving the issue.

Reply:
Why the message queue stuff? No need to update children if there's no path in the data anymore 😉

Reply:
Oops .. forgot about that 😆 Thanks for reminding .. Ill make the changes accordingly!

Reply:
@brusch & @aashan10 Do we really don't need to update children, there is still the "fullpath" column which has to be updated, or am I wrong?

Reply:
@hethehe That is actually a good point! @brusch should we really get rid of the message queue stuff? We removed fullpath from `data` column, however there is still the `fullpath` column which needs to be updated when the user tries to rename the folder.

Reply:
@hethehe totally right, good catch! 
I've created another PR #13417 that should cover this as well. wdyt?